packages/tinacms/src/components/TinaProvider.tsx

```diff
@@ -47,6 +48,10 @@ export const TinaProvider: React.FC<TinaProviderProps> = ({
     })
   }, [])
 
+  if (!(cms instanceof CMS)) {
```

Comment:
I think I would like this to be the stricter check:
```suggestion
  if (!(cms instanceof TinaCMS)) {
```